packages/terra-data-grid/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+* Added
+  * Added `columnHighlightId` to allow column highlighting.
+  * Added `colorHighlightColor` to choose column highlighting colour.
```

Comment:
```suggestion
  * Added `columnHighlightColor` to choose column highlighting colour.
```

Reply:
Removed these and updated log as these properties are moved to column object: https://github.com/cerner/terra-framework/pull/1985/commits/d16d6e95740427f5611a1f2f37e3d81057eb7105